src/pages/vue/your-first-app/5-adding-mobile.md

```diff
@@ -0,0 +1,81 @@
+---
+previousText: 'Loading Photos on Filesystem'
+previousUrl: '/docs/vue/your-first-app/4-loading-photos'
+nextText: 'Deploying Mobile'
+nextUrl: '/docs/vue/your-first-app/6-deploying-mobile'
+---
+
+# Adding Mobile
+
+Our photo gallery app won’t be complete until it runs on iOS, Android, and the web - all using one codebase. All it takes is some small logic changes to support mobile platforms, installing some native tooling, then running the app on a device. Let’s go!
+
+Let’s start with making some small code changes - then our app will "just work" when we deploy it to a device.
+
+## Platform-specific Logic
+
+First, we’ll update the photo saving functionality to support mobile. In the `savePicture` function, check which platform the app is running on. If it’s "hybrid" (Capacitor or Cordova, the two native runtimes), then read the photo file into base64 format using the `readFile` method. Also, return the complete file path to the photo using the Filesystem API. When setting the `webviewPath`, use the special `Capacitor.convertFileSrc` method ([details here](https://capacitorjs.com/docs/basics/utilities#convertfilesrc)). Otherwise, use the same logic as before when running the app on the web.
```

Comment:
```suggestion
First, we’ll update the photo saving functionality to support mobile. In the `savePicture` function, check which platform the app is running on. If it’s "hybrid" (Capacitor, the native runtime), then read the photo file into base64 format using the `readFile` method. Also, return the complete file path to the photo using the Filesystem API. When setting the `webviewPath`, use the special `Capacitor.convertFileSrc` method ([details here](https://capacitorjs.com/docs/basics/utilities#convertfilesrc)). Otherwise, use the same logic as before when running the app on the web.
```

Do we need to mention Cordova here? Ionic Vue doesn't support it